src/index.ts

```diff
@@ -15,3 +19,23 @@ export function convertAxeToSarif(axeResults: Axe.AxeResults): SarifLog {
     const sarifConverter = defaultSarifConverter();
     return sarifConverter.convert(decoratedAxeResults, {});
 }
+
+export function sarifReporter(
+    rawResults: AxeRawResult[],
+    {},
+    callback: Function,
```

Comment:
Would be good to be explicit about this type as well, eg `(sarifResults: SarifLog) => void` instead of just `Function`